### ISSUE TITLE
Panel: defect on Safari

### ISSUE THREAD

**Issue by gcanti** [buildo/react-components#250]
Safari Version 9.0 (9537.86.1.56.2)

From the showcase:

<img width="1105" alt="panel" src="https://cloud.githubusercontent.com/assets/878912/12789022/5ea185ae-ca9c-11e5-8dde-908c76b0c4e8.png">

**Comment by FrancescoCioria** [comment]
I think this may be a duplicate of https://github.com/buildo/react-components/issues/132 but I'll verify it.
thank you

**Comment by FrancescoCioria** [comment]
It looks like Safari requires `flex-basis: auto` to handle correctly the size of children while Chrome/Firefox work also with `flex-basis: 100%` (if you pass only `grow={true}` to `FlexView` the result is `flex: 1 1 100%`)

I think Safari might be right this time, so we're considering to replace the default `100%` with `auto` (after testing it carefully of course)

PS: once #267 is merged `Panel` should behave correctly also on Safari
